libs/novui/src/json-schema-components/JsonSchemaForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { RJSFSchema } from '@rjsf/utils';
+import validator from '@rjsf/validator-ajv8';
+import Form from '@rjsf/core';
+
+import { CheckboxWidget, SelectWidget, InputWidget } from './widgets';
+
+import { AddButton, MoveDownButton, RemoveButton, MoveUpButton } from './templates/IconButton';
+import { ArrayFieldItemTemplate, ArrayFieldTemplate, ArrayFieldTitleTemplate } from './templates/ArrayFieldTemplate';
+import { ObjectFieldTemplate } from './templates/ObjectFieldTemplate';
+
+export const JsonSchemaForm = ({
+  schema,
+  formData,
+  onChange,
+}: {
+  schema: RJSFSchema;
+  formData: any;
+  onChange?: (data: any) => void;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Do we have a way of improving the types here? We should avoid `any` as much as humanly possible‏